app/util/sentry/utils.test.ts

```diff
@@ -126,4 +129,225 @@ describe('captureSentryFeedback', () => {
+            SwapsController: {
+              aggregatorMetadata: 'object',
+              aggregatorMetadataLastFetched: 'number',
+              approvalTransaction: 'object',
+              chainCache: 'object',
+              error: 'object',
+              fetchParams: 'object',
+              fetchParamsMetaData: 'object',
+              isInPolling: 'boolean',
+              pollingCyclesLeft: 'number',
+              quoteRefreshSeconds: 'object',
+              quoteValues: 'object',
+              quotes: 'object',
+              quotesLastFetched: 'number',
+              tokens: 'object',
+              tokensLastFetched: 'number',
+              topAggId: 'object',
+              topAggSavings: 'object',
+              topAssets: 'object',
+              topAssetsLastFetched: 'number',
+              usedCustomGas: 'object',
+              usedGasEstimate: 'object',
+            },
+            TokenBalancesController: 'object',
+            TokenDetectionController: {},
+            TokenListController: {
+              preventPollingOnNetworkRestart: false,
+              tokenList: 'object',
+              tokensChainsCache: {},
+            },
+            TokenRatesController: {
+              marketData: 'object',
+            },
+            TokensController: {
+              allDetectedTokens: {},
+              allIgnoredTokens: {},
+              allTokens: {},
+              detectedTokens: 'object',
+              ignoredTokens: 'object',
+              tokens: 'object',
+            },
+            TransactionController: {
+              lastFetchedBlockNumbers: 'object',
+              methodData: 'object',
+              submitHistory: 'object',
+              transactions: 'object',
+            },
+          },
+        },
+        experimentalSettings: undefined,
+        fiatOrders: 'object',
+        infuraAvailability: undefined,
+        inpageProvider: {
+          networkId: 'loading',
+        },
+        legalNotices: undefined,
+        modals: undefined,
+        navigation: undefined,
+        networkOnboarded: undefined,
+        notification: undefined,
+        onboarding: undefined,
+        privacy: undefined,
+        rpcEvents: undefined,
+        sdk: {
+          approvedHosts: {},
+          connections: {},
+          dappConnections: {},
+        },
+        security: {
+          allowLoginWithRememberMe: false,
+          automaticSecurityChecksEnabled: false,
+          dataCollectionForMarketing: null,
+          hasUserSelectedAutomaticSecurityCheckOption: false,
+          isAutomaticSecurityChecksModalOpen: false,
+          isNFTAutoDetectionModalViewed: false,
+        },
+        settings: undefined,
+        signatureRequest: 'undefined',
+        smartTransactions: {
+          optInModalAppVersionSeen: null,
+        },
+        swaps: 'undefined',
+        transaction: 'undefined',
+        transactionMetrics: 'object',
+        user: {},
+        wizard: undefined,
+      });
```

Comment:
S: as the output you are testing is a big generated JSON, and it's not really useful to have it visible in the test itself (you just copy/pasted the initial return and no one reads each value and removing it would make the test more readable), that's a good use case for using snapshots. Snapshots allow to compare a json, it's not necessarily just for UIs. This seems a good use case for it to me here.

```suggestion
      expect(maskedState).toMatchSnapshot();
```

Reply:
Yeah you are totally right! Addressed on this [commit](https://github.com/MetaMask/metamask-mobile/pull/10241/commits/26ba148811d8c108dfe5ebb850e4a5fe17d9a232)